src/joint.dia.cell.js

```diff
@@ -504,7 +518,25 @@ joint.dia.Cell = Backbone.Model.extend({
     removeAttr: function(path, opt) {
 
         if (_.isArray(path)) {
-            _.each(path, function(p) { this.removeAttr(p, opt); }, this);
+
+            var clone = path.slice();
```

Comment:
Do you have any special reason why you do not use `['attrs'].concat(path)` here? It's not slower (https://jsperf.com/array-unshift-vs-prepend/5) and easier to read IMO.